src/pages/VenuePage/VenuePage.tsx

```diff
@@ -224,21 +351,41 @@ export const VenuePage: React.FC = () => {
 
     if (isEventStartingSoon(event)) {
       return (
-        <CountDown
-          startUtcSeconds={event.start_utc_seconds}
-          textBeforeCountdown="Bar opens in"
-        />
+        <>
+          <Preload venue={venue} />
+          <Suspense fallback={<></>}>
+            <CountDown
+              startUtcSeconds={event.start_utc_seconds}
+              textBeforeCountdown="Bar opens in"
+            />
+          </Suspense>
+        </>
       );
     }
   }
```

Comment:
At this stage i'm not convinced that `Preload` is going to have any meaningful impact on things; and if we do decide to keep it, it shouldn't end up duplicated in so many places.

Reply:
How much do we gain from lazy loading `CountDown` here? I would have thought that it's going to be so small that the extra bundle split wouldn't be worthwhile.

As per previous comment, this should probably be reverted due to conflicts with:

- https://github.com/sparkletown/sparkle/pull/1737

Reply:
It will be easier if https://github.com/sparkletown/sparkle/pull/1737 lands in staging, we do a merge from there I guess.

> At this stage i'm not convinced that Preload is going to have any meaningful impact on things; and if we do decide to keep it, it shouldn't end up duplicated in so many places.

At start, I had the preload component wrap the entire `VenuePage`, but I made it wrap the inner components just so we have the flexibility remove any unnecessary during PR review. 

For `CountDown`, I can run a test with or without it, see if there's a difference. Will add the finding here.

Reply:
> It will be easier if #1737 lands in staging, we do a merge from there I guess.

@mike-lvov Is looking at / merging all of those things now, so should be done soon I expect.

Reply:
> For `CountDown`, I can run a test with or without it, see if there's a difference. Will add the finding here.

Sounds good.

Reply:
`<CountDown>` didn't show noticeable difference in performance between eager and lazy loading in normal venue flow. didn't know how to simulate the event starting soon condition, assuming it's a high load case as well..

Reply:
Removed `Suspense` in https://github.com/sparkletown/sparkle/pull/1713/commits/12ac10cb717639f2354bc2b94a8a56640bde1d4b